config/base.yaml

```diff
@@ -1,4 +1,4 @@
-api-key:
+api_key:
```

Comment:
Can you tell me what has changed here, this is one of the many lines in this PR with no visible changes, but diff highlighted. Could it be that it is some nasty CR/LF/CRLF thing only? If so, we should probably configure our git to keep these under control.

Reply:
I have updated all the configuration file parameters to use underscores instead of dashes as dashes create problems in json and yaml, got some errors when using YQ for creating a new YAML configuration from template.